packages/manager/src/views/DashboardPage/DashboardPage.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  useGetTotalDeployments,
+  useGetDeploymentCounts,
+  useGetDeploymentsTime,
+  useGetMonthlyDeploymentChartWithEphemeral
+} from '@app/services/analytics';
+import { ActivityStream } from './components/ActivityStream';
+import { Analytics } from './components/Analytics';
+
+interface IDeploymentData {
+  env: string;
+  count: number;
+  startDate: string;
+  endDate: string;
+}
+
+export const DashboardPage = (): JSX.Element => {
+  const TotalDeploymentData = useGetTotalDeployments();
+  const Totaldeployment = TotalDeploymentData.data?.reduce((acc, obj) => acc + obj.count, 0);
+  const TotalDeploymentCountsData = useGetDeploymentCounts();
+  const TotalProperty = Object.keys(TotalDeploymentCountsData.data || {}).length;
+  const TotalDeploymentsTimeData = useGetDeploymentsTime();
+  const averageTime = TotalDeploymentsTimeData.data?.averageTime;
+  const TotalMonthlyDeploymentData = useGetMonthlyDeploymentChartWithEphemeral().data;
+  const lastMonthEphemeral = TotalMonthlyDeploymentData?.ephemeral?.reduce(
+    (acc, obj) => acc + obj.count,
+    0
+  );
+
+  const QAData = TotalMonthlyDeploymentData?.qa
```

Comment:
write a custom hook that does this transformation by extending the original hook and applying the selection function required

Ref: https://tkdodo.eu/blog/react-query-data-transformations

Reply:
What about other environments

Reply:
Also can this sorting be done from backend. @soumyadip007  if this is just used for this one

Reply:
We might sort this in backend in future. As of now we can go for this.

Reply:
We are sending all the envs in frontend, but in frontend we'll show the mostly used envs which are Stage, Dev, Prod, QA. Else there will be lot of envs.That is the overall for the entire spaship, you will get the details for your envs in the property page. This chart is of all the dev envs in spaship irrespective of properties